src/Timer_TEST.cc

```diff
@@ -36,6 +36,7 @@ TEST(Timer_TEST, Sequence)
   // Start the timer
   t.Start();
   EXPECT_TRUE(t.Running());
+  std::this_thread::sleep_for(std::chrono::milliseconds(100));
```

Comment:
I found that 1 millisecond was sufficient, but either works.

Reply:
Changed to 1ms to gain time